api.py

```diff
@@ -412,7 +412,124 @@ def delete(self):
         self.flush()
         self.finish()
 
+class MongoHandler(tornado.web.RequestHandler):
+
+    @tornado.web.asynchronous
+    def get(self):
```

Comment:
I don't think we need a get method, just post, its safer while the data is not public